src/nanoemoji/colr_to_svg.py

```diff
@@ -95,7 +96,12 @@ def _draw_svg_path(
 
 
 def _color(ttfont: ttLib.TTFont, palette_index, alpha=1.0) -> colors.Color:
-    cpal_color = ttfont["CPAL"].palettes[0][palette_index]
+    palette = ttfont["CPAL"].palettes[0]
+    if palette_index == _FOREGROUND_COLOR_INDEX:
+        return colors.Color.fromstring("black")  # as good a guess as any
```

Comment:
why black? Should we not use the special "currentColor" here?
https://github.com/googlefonts/nanoemoji/pull/381

Reply:
We should!

Reply:
Tracking resolution of this in https://github.com/googlefonts/nanoemoji/issues/405